test/performance/comm/low-level/many-to-one.chpl

```diff
@@ -49,9 +50,11 @@ proc main() {
   if printTimings then
     timer.start();
```

Comment:
Would you mind getting rid of this timer while you're here?  It's a remnant of the original logic that timed the entire outer coforall across nodes, but I neglected to remove it when I switched to timing the individual tasks.

Reply:
Can do